kratos/python/add_other_utilities_to_python.cpp

```diff
@@ -62,6 +62,77 @@ void SetOnProcessInfo(
     rCurrentProcessInfo[TABLE_UTILITY] = pTable;
 }
 
+double RotateAndCallFunctionCurrentConfiguration(
```

Comment:
as i understand those functions (which are surely interesting) do not really belong in this PR

Reply:
I will check

Reply:
It is used